kube/resources/cron.yaml

```diff
@@ -56,6 +56,11 @@ spec:
                 configMapKeyRef:
                   name: airbyte-env
                   key: WORKSPACE_ROOT
+            - name: MICRONAUT_ENVIRONMENTS
+              valueFrom:
+                configMapKeyRef:
+                  name: airbyte-env
+                  key: WORKERS_MICRONAUT_ENVIRONMENTS
```

Comment:
We should have a different name for the Cron environments in case it ends up having different environments than workers.

Reply:
ok, I will do it. I am wondering how well this will work on cloud.

Reply:
Done.